content/dstu2/encounter.md

```diff
@@ -7,18 +7,31 @@ title: Encounter | DSTU 2 API
 * TOC
 {:toc}
 
+## Terminology Bindings
+
+<%= terminology_table(:encounter, :dstu2) %>
+
+### Contained Location Bindings
+
+<%= terminology_table(:location, :dstu2) %>
+
 ## Search
 
 Search for Encounters that meet supplied query parameters:
 
     GET /Encounter?:parameters
 
+_Implementation Notes_
+
+* The [hospitalization.destination] will be returned as a reference to a [contained] location resource.
```

Comment:
Can these be Encounter.hospitalization.destination and Encounter.location.location since that is how the linked fields are formatted?